dataflow/src/main/java/org/checkerframework/dataflow/cfg/block/Block.java

```diff
@@ -41,4 +43,18 @@
      * @return the predecessors of this basic block
      */
     Set<Block> getPredecessors();
+
+    /**
+     * Returns the successors of this basic block.
+     *
+     * @return the successors of this basic block
+     */
+    Set<Block> getSuccessors();
+
+    /**
+     * Returns the nodes contained within this basic block.
```

Comment:
At the moment there is no documentation which blocks return something non-empty or with just a single element. Maybe documenting that would be useful with the individual implementations of this method.

Reply:
I added that documentation.